storage/src/lib.rs

```diff
@@ -104,7 +110,7 @@ pub type Result<T> = result::Result<T, Error>;
 
 pub struct Storage {
     pub config: StorageConfig,
-    lookups: BTreeMap<PackHash, indexfile::Lookup>,
+    pub lookups: LinkedHashMap<PackHash, indexfile::Lookup>,
 }
```

Comment:
1. `BTreeMap` is replaced with `LinkedHashMap` in order to preserve the insertion order, cuz we need epoch indexes to be in the "chain order" for block-by-height query to work.
2. `pub` were added only for experiments/debug, I will remove it